File: baekjoon/01-입출력/js/11720.js
// 백준 제출 시, readFileSync('/dev/stdin') 으로 변경할 것
const fs = require("fs");
const input = fs.readFileSync("예제.txt").toString().split("\n");

const n = input[0];
const numbers = input[1];

let sum = 0;
for (let i of numbers) {
  sum += Number(i);
}

console.log(sum);

File: baekjoon/01-입출력/c++/11720.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    int n, sum = 0;
    string numbers;

    cin >> n;
    cin >> numbers;
    for (int i = 0; i < n; i++) {
        int int_value = numbers[i] - '0';
        sum += int_value;
    }
    cout << sum <<endl;
}